test/redis_test.exs

```diff
@@ -96,4 +96,26 @@ defmodule Exq.RedisTest do
     Connection.flushdb! :testredis
     assert Connection.smembers!(:testredis, "theset") == []
   end
+
+  test "prepare_script" do
```

Comment:
can we add a test that checks for order of jobs re_enqueued from backup?  Comparing ones that are in backup queue, and also ones in current queue, after re-enqueue it should be in same order as originally queued.